templates/authorizing.html

```diff
@@ -0,0 +1,82 @@
+{#-#################################################################
+<File Name>
+  authorizing.html
```

Comment:
I feel like `authorizing` is not a good name for the template. What about `functionary_selection` or something along those lines.

Reply:
Hm. Matter of taste. What about `authorizing_functionaries`? :) I'm fine either way.

Reply:
sure, `authorizing_functionaries` sounds good in my book :)